src/closest-polyfill.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Element.closest polyfill
```

Comment:
I'm not sure this is strictly necessary, I believe we get this already from Polyfill.io?

Reply:
Since this is an open source standalone library, I don't think we should rely on consuming applications having the polyfill.